src/wp-includes/rest-api/endpoints/class-wp-rest-font-library-controller.php

```diff
@@ -0,0 +1,556 @@
+				__( 'No fonts to uninstall' )
+			);
+			$data     = array(
+				'successes' => $successes,
+				'errors'    => $errors,
+			);
+			$response = rest_ensure_response( $data );
+			$response->set_status( 400 );
+			return $response;
+		}
+
+		foreach ( $fonts_to_uninstall as $font_data ) {
+			$font   = new WP_Font_Family( $font_data );
+			$result = $font->uninstall();
+			if ( is_wp_error( $result ) ) {
+				$errors[] = $result;
+			} else {
+				$successes[] = $result;
+			}
+		}
+		$data = array(
+			'successes' => $successes,
+			'errors'    => $errors,
+		);
+		return rest_ensure_response( $data );
+	}
+
+	/**
+	 * Checks whether the user has permissions to update the Font Library.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return true|WP_Error True if the request has write access for the item, WP_Error object otherwise.
+	 */
+	public function update_font_library_permissions_check() {
+		if ( ! current_user_can( 'edit_theme_options' ) ) {
+			return new WP_Error(
+				'rest_cannot_update_font_library',
+				__( 'Sorry, you are not allowed to update the Font Library on this site.' ),
+				array(
+					'status' => rest_authorization_required_code(),
+				)
+			);
+		}
+		return true;
+	}
+
+	/**
+	 * Retrieves the item's schema, conforming to JSON Schema.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'font-collection',
+			'type'       => 'object',
+			'properties' => array(
+				'id'          => array(
+					'description' => __( 'Unique identifier for the font collection.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'name'        => array(
+					'description' => __( 'Name of the font collection.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+				),
+				'description' => array(
+					'description' => __( 'Description of the font collection.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+				),
+				'data'        => array(
+					'description' => __( 'Data of the font collection.' ),
+					'type'        => 'object',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'properties'  => array(
+						'fontFamilies' => array(
+							'description' => __( 'List of font families.' ),
+							'type'        => 'array',
+							'items'       => array(
+								'type'       => 'object',
+								'properties' => array(
+									'name'       => array(
+										'description' => __( 'Name of the font family.' ),
+										'type'        => 'string',
+									),
+									'fontFamily' => array(
+										'description' => __( 'Font family string.' ),
+										'type'        => 'string',
+									),
```

Comment:
Query: What's the difference between `name` and `fontFamily`?

Reply:
I think (correct me if I'm wrong @matiasbenedetto) both align to `theme.json`'s structure of `settings.typography.fontFamilies` fields.

For example, [TT4 uses](https://github.com/WordPress/twentytwentyfour/blob/be827b8e3a5932191511c033d73a2898e110bed5/theme.json#L200-L214):
```
{
	"fontFace": [
		{
			"fontFamily": "Inter",
			"fontStretch": "normal",
			"fontStyle": "normal",
			"fontWeight": "300 900",
			"src": [
				"file:./assets/fonts/inter/Inter-VariableFont_slnt,wght.woff2"
			]
		}
	],
	"fontFamily": "\"Inter\", sans-serif",
	"name": "Inter",
	"slug": "body"
},
```

Reply:
yep, correct.

Reply:
@costdev using the theme.json schema / structure, the 2 fields are different in that the `fontFamily` field could be multiple fonts that are comma-separated (think in terms of CSS).

Reply:
Thanks both! It may be worth clarifying that in the `description` key of the fields. [See the theme.json schema](https://github.com/WordPress/gutenberg/blob/trunk/schemas/json/theme.json#L576-L588). Also note the description of `fontFace.fontFamily` on line 595.